index.html

```diff
@@ -248,17 +316,23 @@ <h1>Get started</h1>
                 <label for="email">Your email</label>
                 <input
                   type="email"
+                  onchange="storeData();"
                   id="email"
                   name="email"
                   placeholder="name@email.com"
                   required
+                  pattern="[a-z@.]+"
                 />
               </li>
             </ul>
+            <script src="valid.js"></script>
```

Comment:
- [ ] Kindly move your script to the button of the body or header section. It's bad practice to add the script in the middle of your HTML code.